provider/pkg/naming/convert.go

```diff
@@ -179,7 +180,12 @@ func (c *sdkToCfnConverter) sdkObjectValueToCfn(typeName string, spec metadata.C
 
 func (c *sdkToCfnConverter) diffToPatch(diff *resource.ObjectDiff) ([]jsonpatch.JsonPatchOperation, error) {
 	var ops []jsonpatch.JsonPatchOperation
-	for sdkName, prop := range c.spec.Inputs {
+	inputKeys := codegen.NewStringSet()
+	for sdkName := range c.spec.Inputs {
+		inputKeys.Add(string(sdkName))
+	}
+	for _, sdkName := range inputKeys.SortedValues() {
+		prop := c.spec.Inputs[sdkName]
```

Comment:
The diffs weren't stable because we're iterating a map. This wasn't an issue until I was trying to write asserts in the unit test. Though, on second look, we could probably swap the `StringSet` for a string array which we just call `strings.SortStable` on.

Reply:
ok, might be nice to add a comment to that effect "sort keys to stabilize diff ordering"